src/wp-content/themes/twentytwentyone/inc/template-functions.php

```diff
@@ -17,9 +17,6 @@
  */
 function twenty_twenty_one_body_classes( $classes ) {
 
-	// Helps detect if JS is enabled or not.
-	$classes[] = 'no-js';
```

Comment:
This has been moved to the `html` element for consistency with other themes. It is more performant on the document element, since if the class is modified on the `body` element, then this introduces the possibility of there being a re-layout due to a style change.

Reply:
See above, this is unrelated to this ticket/PR.